extensions.md

```
@@ -0,0 +1,282 @@
+```bash
+ns extension install <package-name>
+ns extension uninstall <package-name>
+```
+
+Installed extensions live in the CLI profile directory, under
+`extensions/node_modules/<package-name>`, and every CLI invocation consults each
+of them. That makes the manifest below the most important file in an extension:
+it is what the CLI reads on startup, and it decides whether your code is loaded
+eagerly or only when one of your commands is actually executed.
+
+## Depending on the CLI
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)